Newsletters: add 269 (2023-09-20)

harding opened the pull request:
- [x] (Monday) Notable code changes @harding 
- [x] (Tuesday) Lede, update releases/RCs, add topic entries @harding 
- [x] Clients/Services section @bitschmidty 

Note: I would have liked to have covered John Law's LN with covenants proposal from last week in this newsletter, but I'm wrapping up some travel with family and ran out of time to give it a proper treatment.  I'll give it the attention it deserves in next week's newsletter.

vostrnad commented on _posts/en/newsletters/2023-09-20-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+
+## News
+
+- **Bitcoin research event:** Sergi Delgato Segura and Clara Shikhelman
```
```suggestion
- **Bitcoin research event:** Sergi Delgado Segura and Clara Shikhelman
```

LarryRuane commented on _posts/en/newsletters/2023-09-20-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+  to continue to access blocks and transactions in stripped format
+  (without any segwit fields).  At this point, all programs should be
+  updated to handle segwit transactions and this option should no longer
+  be needed, although in can be temporarily re-enabled as a deprecated
```
```suggestion
  be needed, although it can be temporarily re-enabled as a deprecated
```

LarryRuane commented on _posts/en/newsletters/2023-09-20-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+  allows an operator to manually access the same ability.
+
+- [LDK #2176][] increases the precision with which LDK attempts to
+  probabalistically guess the amount of liquidity available in distant
```
```suggestion
  probabilistically guess the amount of liquidity available in distant
```

harding commented:
Made edits for feedback, thanks @vostrnad and @LarryRuane!  Reviewed @bitschmidty's contribution (thanks!).  Added lede, updated releases/RCs, and added topic links.

Thanks everyone!

bitschmidty commented on _posts/en/newsletters/2023-09-20-newsletter.md:
```diff
@@ -0,0 +1,159 @@
+
+- **Machankura announces additive batching feature:**
+  Bitcoin service provider [Machankura][] has [announced][machankura tweet] a
+  beta feature that supports additive [batching] using RBF in a [taproot][topic
```
Ah, that was actually unintentional, thanks for catching those.

bitschmidty commented:
Made a small fixup "fee deficient"->"fee deficit", squashed and merged, thank you all! 🚀
